src/util_libc.rs

```diff
@@ -26,6 +26,8 @@ cfg_if! {
         use libc::__error as errno_location;
     } else if #[cfg(target_os = "haiku")] {
         use libc::_errnop as errno_location;
+    } else if #[cfg(target_os = "nto")] {
+        use libc::__get_errno_ptr as errno_location;
```

Comment:
Out of curiosity, where is this documented?

Reply:
AFAIK this is not documented but `errno.h` of the QNX/nto toolchain uses it for `errno` as well:
```c
# define errno (*__get_errno_ptr())
```